mitmproxy/proxy/mode_servers.py

```diff
@@ -12,6 +12,10 @@
 from __future__ import annotations
 
 import asyncio
+import json
+import textwrap
+from pathlib import Path
+
 import errno
```

Comment:
Looks like the new imports are not sorted correctly?

Reply:
We currently don't insist on a particular import style or order, we only use flake8 to flag the most egregious style violations. The reason for this is that it doesn't provide much value in itself while often getting in the way of merging PRs. It's not a very common thing for Python projects (Rust is very different here), so new contributors are unlikely to sort imports right away.